addons/account/models/company.py

```diff
@@ -78,12 +78,12 @@ class ResCompany(models.Model):
     currency_exchange_journal_id = fields.Many2one('account.journal', string="Exchange Gain or Loss Journal", domain=[('type', '=', 'general')])
     income_currency_exchange_account_id = fields.Many2one(
         comodel_name='account.account',
-        string="Gain Exchange Rate Account",
+        string="Gain",
```

Comment:
I think it's because

Reply:
I think it's because it's clear enough here in the settings:

![image](https://github.com/odoo/odoo/assets/91618364/1259bc71-2ab0-4420-baa8-911931805bb9)

But then I will update the string in the xml and keep the full string on the field

Reply:
The intention of the spec is to adapt the res.config.settings view and make labels shorter by removing the redundancy of the word "account".

Reply:
"Journal", "Gain Account" and "Loss Account" don't seem really redundant to me :thinking: